managedtenants/data/selectorsyncset.yaml.j2

```diff
@@ -66,7 +66,7 @@ items:
       kind: CatalogSource
       metadata:
         name: addon-{{ADDON.metadata['id']}}-catalog
-        namespace: openshift-marketplace
+        namespace: {{ADDON.metadata["targetNamespace"]}}
```

Comment:
we also have to change the `sourceNamespace` reference to this CatalogSource in the Subscription